docs: Update the Cloud Hypervisor description in virtualization.md

likebreath opened the pull request:
The current description on the Cloud Hypervisor support in Kata
containers were introduced back to kata 1.10 and are out-dated.

Depends-on: github.com/kata-containers/tests#3106

Fixes: #1167

Signed-off-by: Bo Chen <chen.bo@intel.com>

devimc commented on docs/design/virtualization.md:
```diff
@@ -100,23 +100,33 @@ Devices used:
 - virtio fs
+- VFIO
+- hotplug
+- seccomp filters
```
and pmem ?

likebreath replied:
Good point. Just added. Thanks!

devimc reviewed (approved):
thanks @likebreath - one question

egernst commented:
Looks like you'll need to battle with travis over spelling, formatting.  PTAL @likebreath

likebreath commented:
@egernst Thanks for pointing out that. Just added a new PR in the tests repo to address the spelling issue and also added a `Depends-on` in the commit message. Not sure whether it will work for Travis-CI though. PTAL a look at: kata-containers/tests#3106. Thanks!

likebreath commented:
/test
